templates/circuit.yaml

```diff
@@ -227,3 +227,28 @@ spec:
         artifacts:
           - name: extended-circuit
             path: /app/extended_circuit.json
+
+    # Combine circuits in a circuitset to create a composite circuit
+    - name: combine-circuits
```

Comment:
Let's rename this to `concatenate-circuits` to avoid potential confustion, but besides that it looks great!

Reply:
Addressed in [this commit](https://github.com/zapatacomputing/z-quantum-core/pull/82/commits/e5849cfe0728839d47580356aaa0929c2ec01836)